README.md

```diff
@@ -273,13 +280,17 @@ This repository contains a curated list of awesome open source libraries that wi
 * [KNative Serving](https://github.com/knative/serving) ![](https://img.shields.io/github/stars/knative/serving.svg?style=social) - Kubernetes based serverless microservices with "scale-to-zero" functionality.
 
 ## Computation load distribution frameworks
+* [Ray](https://github.com/ray-project/ray) ![](https://img.shields.io/github/stars/ray-project/ray.svg?style=social) - Ray is a flexible, high-performance distributed execution framework for machine learning ([VIDEO](https://www.youtube.com/watch?v=D_oz7E4v-U0))
```

Comment:
Ray is already in Model Deployment and Orchestration

Reply:
The reason for this change was to stress the fact that Ray can be used for generic distributed computing, not just for ML-related tasks like hyper-parameter tuning.
Based on my experience with Ray it has very little to do with "Model Deployment and Orchestration Frameworks" because Ray is mostly a research tool, not a serving engine.
I can revert this change if you insisnt though...